timesheet_task/__openerp__.py

```diff
@@ -23,7 +23,7 @@
  'author': "Camptocamp,Odoo Community Association (OCA)",
  'maintainer': 'Camptocamp, Acsone SA/NV',
  'category': 'Human Resources',
- 'depends': ['project', 'hr_timesheet_invoice'],
+ 'depends': ['project', 'analytic_user_function'],
```

Comment:
Better to use then `web_context_tunnel` for that signature change: https://github.com/OCA/server-tools/tree/8.0/web_context_tunnel